code/game/objects/items/theft_tools.dm

```diff
@@ -212,8 +212,13 @@
 
 /obj/item/nuke_core/supermatter_sliver/pickup(mob/living/user)
 	..()
+	var/mob/living/carbon/carbon_user = user
 	if(!isliving(user) || user.status_flags & GODMODE) //try to keep this in sync with supermatter's consume fail conditions
 		return FALSE
+	if(istype(carbon_user.gloves, /obj/item/clothing/gloves/nob_gloves))
+		user.visible_message(span_notice("[user] reaches out and tries to pick up [src], the gloves protecting them from any danger."),\
+			span_userdanger("You reach out for the [src], the gloves protecting you from any potential harm."))
+		return
```

Comment:
This should be a trait the gloves provide (Look at how clumsy works for insuls), rather then typechecking for the gloves constantly. Maybe call it SUPERMATTER_IMMUNE? Or prehaps something more glove specific, since you want the message to mention hands

Regardless, typechecks like this are kinda fragile, and I don't like em

Reply:
fair point, ill look into this soon enough. (it is 2 am)